src/styles/index.scss

```diff
@@ -1,3 +1,82 @@
 body {
   margin: 0;
+  padding: 0;
+}
+
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.stopwatch {
+  height: 80vmin;
+  width: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+  position: relative;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &__minute-hand {
+    width: 3vmin;
+    height: 20vmin;
+    background-color: #0700ff;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -100%);
+    transform-origin: bottom;
+    z-index: -1;
+    animation: move-minute-hand 3600s steps(60) infinite;
+  }
+
+  &__second-hand {
+    width: 1.5vmin;
+    height: 38vmin;
+    background-color: #2c8000;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -100%);
+    transform-origin: bottom;
+    z-index: -2;
+    animation: move-second-hand 60s linear infinite;
+  }
+
+  &__center {
+    height: 5vmin;
+    width: 5vmin;
+    background-color: #f6a603;
+    border-radius: 50%;
+  }
+
+  &--speed-up &__minute-hand {
+    animation: move-minute-hand 600s steps(60) infinite;
+  }
+
+  &--speed-up &__second-hand {
+    animation: move-second-hand 10s linear infinite;
```

Comment:
The modifier `&--speed-up` is used here to increase the animation speed. However, ensure that the modifier is correctly applied to the parent element in the HTML structure. If the modifier is not applied to the `.stopwatch` element, these styles will not take effect.